src/metrics/include/metrics/operation.hpp

```diff
@@ -197,14 +223,19 @@ class OperationImpl final : private boost::noncopyable {
      * @return the time series for the operation being run.
      */
     const EventSeries& getEvents() const {
-        return _events;
+        return *_events;
     }
 
     void reportAt(time_point started, time_point finished, OperationEventT<ClockSource>&& event) {
         if (_threshold) {
             _threshold->check(started, finished);
         }
-        _events.addAt(finished, event);
+        if (_registry.getFormat().use_grpc()) {
+            _stream->addAt(finished, event, _registry.getWorkerCount(_actorName, _opName));
```

Comment:
this never changes after we're done with setup - is there a way to pass this value once rather than continuously asking the registry?

Reply:
bump here - the workerCount shouldn't ever change post cosntruciton

Reply:
Ahh we'd figured out in the last ticket (where the `getWorkerCount` function was added), that since the actors are constructed in sequence then the earlier `OperationImpl` constructions don't know what the final number of workers added to the phase will be.